homeassistant/components/pterodactyl/api.py

```diff
@@ -43,6 +44,15 @@ class PterodactylData:
     uptime: int
 
 
+class PterodactylCommands(StrEnum):
+    """Command enums for the Pterodactyl server."""
```

Comment:
```suggestion
class PterodactylCommand(StrEnum):
    """Command enum for the Pterodactyl server."""
```
In theory this would be the most pythonic naming style for this